hera_opm/mf_tools.py

```diff
@@ -758,46 +980,53 @@ def build_analysis_makeflow_from_config(
                 print("export BATCH_OPTIONS = {}".format(batch_options), file=f)
 
                 # make rules
-                for pol, outfile in zip(pol_list, outfiles):
-                    time_prereqs = get_config_entry(
-                        config, action, "time_prereqs", required=False
+                for outfile in outfiles:
+                    prereqs = get_config_entry(
+                        config, action, "prereqs", required=False
                     )
-                    if time_prereqs is not None:
-                        if not isinstance(time_prereqs, list):
-                            time_prereqs = [time_prereqs]
+                    if prereqs is not None:
+                        if not isinstance(prereqs, list):
+                            prereqs = [prereqs]
                         # get how many neighbors we should be including
-                        n_neighbors = get_config_entry(
-                            config, action, "n_time_neighbors", required=True
+                        n_time_neighbors = get_config_entry(
+                            config,
+                            action,
+                            "n_time_neighbors",
+                            required=False,
+                            total_length=len(obsids),
+                        )
+                        time_centered = get_config_entry(
+                            config, action, "time_centered", required=False
                         )
 
-                        # get a copy of the infile list; we're going to add to it, but don't want these
-                        # entries broadcast across pols
-                        infiles_pol = infiles
-                        for tp in time_prereqs:
+                        for tp in prereqs:
```

Comment:
I think the dummy variable `tp` doesn't make sense here given that we've changed the name of the list... maybe `pr`?